utils/cryptodev-linux/Makefile

```diff
@@ -14,7 +14,7 @@ PKG_VERSION:=1.7
 PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://download.gna.org/cryptodev-linux/
+PKG_SOURCE_URL:=http://nwl.cc/pub/cryptodev-linux/
```

Comment:
download.gna.org worked yesterday (pretty sure), add both while keeping gna.org as primary source

Reply:
Done, thank you! Now still down by the way.